python/.archivessnake.yml

```diff
@@ -0,0 +1,5 @@
+baseurl: http://archivesspace:8089
+username: admin
+password: admin
+logging_config:
+default_config: DEBUG_TO_FILE
```

Comment:
The [documentation](https://github.com/archivesspace-labs/ArchivesSnake?tab=readme-ov-file#configuration) does not make this very clear... but I believe the `default_config:` line needs to be indented, because it's a
> Hash with various config values for the logging subsystem

As-is, when I try to instantiate this, I get this error (this is with the local system fully started, and the `admin` password set to `admin`):
```
$ docker compose run python python
Python 3.11.9 (main, Aug 13 2024, 01:17:41) [GCC 10.2.1 20210110] on linux
Type "help", "copyright", "credits" or "license" for more information.

>>> from asnake.aspace import ASpace
>>> a = ASpace()

Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/aspace/.local/lib/python3.11/site-packages/asnake/aspace/__init__.py", line 15, in __init__
    self.client = ASnakeClient(**config)
                  ^^^^^^^^^^^^^^^^^^^^^^
  File "/home/aspace/.local/lib/python3.11/site-packages/asnake/client/web_client.py", line 73, in __init__
    if 'default_config' in self.config['logging_config']:
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: argument of type 'NoneType' is not iterable
```

As-is, the `logging_config` key has no value assigned (`None`), because there's nothing else on that line in the yaml file, and nothing indented below it to indicate it has any data.

When I indent line 5 by 2 spaces, I'm able to instantiate without errors, and can connect to my local instance.